README.md

```diff
@@ -192,6 +193,9 @@ keypair_path: ~/.ssh/mykeypair.pem
 #
 # Be sure to wrap its value in single quotes.
 pull_secret: ''
+
+# Install the cluster in FIPS mode 
+install_fips_mode: false 
```

Comment:
Personal preference, but can you name the variable `fips_mode` instead of `install_fips_mode`?